website/_config.yml

```diff
@@ -0,0 +1,19 @@
+title: Krayon
+description: Cross platform (Android, iOS & JavaScript) canvas library.
+
+plugins:
+  - jekyll-remote-theme
+  - jekyll-spaceship
+
+remote_theme: jekyll/minima
+
+kramdown:
+  parse_block_html: true
+
+jekyll-spaceship:
+  processors:
+    - mermaid-processor
```

Comment:
Jekyll Starships supports many processors, but only enabled Mermaid, as one of the other processors broke some of the markdown rendering.